modules/migrationmngr/keeper/abci.go

```diff
@@ -38,6 +38,24 @@ func (k Keeper) PreBlock(ctx context.Context) (appmodule.ResponsePreBlock, error
 
 	// one block after the migration end, we halt forcing a binary switch
 	if shouldHalt {
+		migration, err := k.Migration.Get(ctx)
+		if err != nil {
+			k.Logger(ctx).Error("failed to get migration state", "error", err)
+			return nil, sdkerrors.ErrLogic.Wrapf("failed to get migration state: %v", err)
+		}
+
+		if migration.StayOnComet {
```

Comment:
main change, this field just means we fall back on staking module to handle validator updates and don't halt the chain.